### PR TITLE
Relax runtime requirments

### PR DESCRIPTION
**Pull request by isuruf**
This changes `clangdev 4.0.0` to `clangdev 4.0.0|4.0.0.*`

Needs this for packaging Flang

### CODE REVIEW

**Comment by isuruf** [comment]
@jakirkham, ping

**Comment by isuruf** [comment]
Ping

File: recipe/meta.yaml [hunk 1]
```diff
@@ -57,3 +57,4 @@ extra:
     - inducer
     - jakirkham
     - yesimon
+    - isuruf
```
Comment (inducer) [inline recipe/meta.yaml]: This should *kind of* be a separate PR, right? :)

Comment (isuruf) [reply recipe/meta.yaml]: Yeah. sorry about that.
